### ISSUE TITLE
Implement additional restic functionality

### ISSUE THREAD

**Issue by YannikBramkamp** [mittwald/brudi#19]
Make already existing functions such as `ListSnapshots` available via flags or commands.

**Comment by YannikBramkamp** [comment]
Current state:

Additional flags:
- `--restic-check`:
   Checks repository integrity with `restic check`. Should be useful 
- `--restic-prune`:
   Removes any unneeded data left by for example `restic forget` with `restic prune`. Could be run after `--forget`
- `--restic-rebuild-index`:
   Rebuilds repository index with `restic rebuild-index`. Potentially useful, although maybe not as a flag that is run after `restic backup`?
- `--restic-snapshots`:
   List snapshots in repository with `restic snapshots`. Potentially useful, but maybe more in conjunction with a "blank" command? 
- `--restic-tags`:
   Edit tags of snapshots with `restic tag`. Limited usefulness, especially es a flag.

In summary, as far as I can see, the first three flags would be actually useful as a flag, while `--restic-tags` and  `--restic-snapshots` would be more limited in usefulness. Would it be worth considering to create a "maintenance" command that could be used to run those flags?
